ord_schema/message_helpers.py

```diff
@@ -386,6 +389,11 @@ def get_reaction_smiles(
 
     reactants, agents, products = set(), set(), set()
     roles = reaction_pb2.ReactionRole
+    reactant_roles = [roles.REACTANT]
+    product_roles = [roles.PRODUCT]
```

Comment:
Is it intentional to exclude BYPRODUCT or SIDE_PRODUCT?

Reply:
Yes; we are only including the desired product when generating reaction SMILES. Happy to discuss if you think we should change that, but maybe in a new issue?